dpdata/system.py

```diff
@@ -229,7 +229,12 @@ def from_fmt_obj(self, fmtobj, file_name, **kwargs):
                 self.data = {**self.data, **data}
                 self.check_data()
             if hasattr(fmtobj.from_system, "post_func"):
+                post_func_skip_list = kwargs.get("post_func_skip_list", [])
```

Comment:
The argument should be written explicitly, otherwise others won't realize this argument.